packages/terra-tabs/src/clinical-lowlight-theme/Tabs.module.scss

```diff
@@ -0,0 +1,117 @@
+:local {
+  .clinical-lowlight-theme {
+    --terra-tabs-keyboard-focus-ltr-box-shadow: 0 0 1px 3px #4a90e2, 0 0 7px 4px #4a90e2;
```

Comment:
note: when the theme variables are organized differently in each theme file, it makes it quite difficult on reviewers to ensure that all theme variables are covered.